Rob_Params:
  num_rob: 2
  sph0: sphero_gbr
  sph1: sphero_yyw

Blob_Params:
  threshold_step: 10
  min_threshold: 45
  max_threshold: 255
  min_repeatability: 2
  min_dist_between_blobs: 900 #only one blob should be detected
  filter_by_color: false
  blob_color: 0
  filter_by_area: true
  min_area: 10
  max_area: 10000
  filter_by_circularity: true
  min_circularity: 0.3
  max_circularity: 100000
  filter_by_inertia: false
  min_inertia_ratio: 0.5
  max_inertia_ratio: 100
  filter_by_convexity: false
  min_convexity: 0.95
  max_convexity: 100
